### DISCUSSION TITLE
Meeting 2024-08-20

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#788] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-08-20)

### Announcements

* None yet

**Comment by eldruin** [comment]
1. Embedded Rust Survey: #787 

2. Is anyone attending GH Universe and want to showcase embedded Rust?
> We love open source projects - and we appreciate all that maintainers do to keep the software ecosystem thriving. At [GitHub Universe](https://githubuniverse.com/) in San Francisco (October 29-30, 2024), we're looking forward to showcasing some open source projects with dedicated community tables.
> 
> Are you interested in showcasing your project at GitHub Universe in our open source zone? We'll give you two free tickets for Universe. Just bring yourself and a laptop!
> 
> We have limited spots, so signing up is an indication of interest, but we can't guarantee all projects a spot. We'll reach out and let you know soon. We also have some limited travel budget. Applications close August 24th 2024.
> 
> **Let us know if you're interested here:** https://gh.io/opensourcelounge

**Reply by jamesmunns** [reply]
Outstanding questions to discuss during the meeting:

Outstanding questions to discuss in the meeting:

* [x] Do we want to enumerate "popular crates" to seed the list? (see [this comment](https://github.com/rust-embedded/wg/pull/787#discussion_r1720718252) )
* [x] Do we want to enumerate "common blockers" to seed the list? (see [this comment](https://github.com/rust-embedded/wg/pull/787#discussion_r1720718866) )
* [x] Do we want to add a question asking what other languages are used in embedded Rust projects? (see [this comment](https://github.com/rust-embedded/wg/pull/787#discussion_r1721093403) )

**Comment by jamesmunns** [comment]
Making a thread to discuss where we should promote the WG survey over the next two weeks:

* Blogs
    * [Inside Rust](https://blog.rust-lang.org/inside-rust/) blog 
    * [Interrupt](https://interrupt.memfault.com/) blog from Memfault
* Newsletters
    * The [Embedded Rustacean](https://www.theembeddedrustacean.com/)
* Podcasts
    * [Self-Directed Research](https://sdr-podcast.com/)
    * [Oxide and Friends](https://oxide-and-friends.transistor.fm/)
* Mailing lists
    * [Tock-OS](https://lists.tockos.org/postorius/lists/devel.lists.tockos.org/)
    * Rust internal team mailing list

**Reply by theembeddedrustacean** [reply]
Awesome! A broadcast has been scheduled on The Embedded Rustacean for this Monday to avoid the weekend. A special announcement will also be included in the upcoming edition of the newsletter, which reaches approximately 3k subscribers.

**Reply by jamesmunns** [reply]
@theembeddedrustacean thank you!

**Comment by BartMassey** [comment]
# ~~Book Sprint!~~ cancelled

* When: ~~This Saturday 24 August, 0830-1230 US Pacific Time (GMT - 7)~~
* Where: https://pdx.zoom.us/j/84919326368
* What: We'll be working on a chapter for the Embedded Discovery Book, to be determined. Probably DMA and some interrupt stuff.
* Conditions: Four of you add yourselves here as participating. (A book sprint needs people.)

Book sprints are fun. We'll edit an outline I'll provide, then dive in to write content. If you have to come late or leave early, that's great. We probably won't finish in the time, so be prepared to have a bit of homework. I will edit the final result.

Thanks for your help with the book!

**Reply by jamesmunns** [reply]
I'll be there for at least part of it!

**Reply by adamgreig** [reply]
I can be there for probably all of it, but possibly joining a bit late... I'll have a better idea closer to the time.

**Reply by BartMassey** [reply]
@adamgreig @jamesmunns thanks much for your kind offer of help. Let's try again for an October date and see if we get a bigger crowd.
